### DISCUSSION TITLE
I think we need something to sync stores across tabs

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#115] [Ideas]
I think we need something to sync stores across tabs.

What do you think?

**Comment by jycouet** [comment]
Hummm, not sure to know what you want to do with this?
You can already have `localstorage cache` that is shared across tabs.

Could you explain a bit more?

**Reply by frederikhors** [reply]
If I update a store in a tab the other tab is not updated.

Other libraries are struggling on this too: https://github.com/FormidableLabs/urql/issues/1062.
